frontend/packages/dev-console/src/utils/pipeline-actions.tsx

```diff
@@ -135,7 +135,7 @@ const rerunPipeline: KebabAction = (
 ) => {
   const { onComplete } = customData;
 
-  const sharedProps = { label: 'Start Last Run', accessReview: {} };
+  const sharedProps = { label: 'Rerun', accessReview: {} };
```

Comment:
Yeah @divyanshiGupta is right, this broke a test... please update the test as we are wanting to change this to `Rerun` for all cases that use re-run of a PipelineRun.